datajoint/external.py

```diff
@@ -87,24 +100,38 @@ def get(self, hash):
         """
         store = hash[STORE_HASH_LENGTH:]
         store = 'external' + ('-' if store else '') + store
-        try:
-            spec = config[store]
-        except KeyError:
-            raise DataJointError('Store `%s` is not configured' % store)
+        cache_file = os.path.join(config['cache'], hash) if 'cache' in config else None
```

Comment:
I also think that we should skip caching if `config['cache'] = None` (or any falsy values e.g. `False`), so change this to `... if 'cache' in config and config['cache'] else None`

Reply:
agreed